### ISSUE TITLE
[JList] NullPointerException when calculating baseline.

### ISSUE THREAD

**Issue by Ilolka1337** [weisJ/darklaf#150]
**Additional Information:**
 - OS: [Win]
 - OS Version: [10]
- Darklaf Version: [2.0.3]

**Additional context**
Log:
[log.txt](https://github.com/weisJ/darklaf/files/4533137/log.txt)
````
java.lang.NullPointerException
	at javax.swing.plaf.basic.BasicListUI.getBaseline(Unknown Source)
	at com.github.weisj.darklaf.ui.list.DarkListUIBridge.getBaseline(DarkListUIBridge.java:356)
	at javax.swing.JComponent.getBaseline(Unknown Source)
	at javax.swing.plaf.basic.BasicScrollPaneUI.getBaseline(Unknown Source)
	at javax.swing.JComponent.getBaseline(Unknown Source)
	at javax.swing.GroupLayout$ComponentSpring.getBaseline(Unknown Source)
	at javax.swing.GroupLayout$BaselineGroup.calculateBaselineAndResizeBehavior(Unknown Source)
	at javax.swing.GroupLayout$BaselineGroup.calculateSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculatePreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Spring.getPreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Group.getSpringSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculateSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculatePreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Spring.getPreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Group.getSpringSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculateSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculatePreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Spring.getPreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Group.getSpringSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculateSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculatePreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Spring.getPreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Group.getSpringSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculateSize(Unknown Source)
	at javax.swing.GroupLayout$Group.calculatePreferredSize(Unknown Source)
	at javax.swing.GroupLayout$Spring.getPreferredSize(Unknown Source)
	at javax.swing.GroupLayout$SequentialGroup.setValidSize(Unknown Source)
	at javax.swing.GroupLayout$Group.setSize(Unknown Source)
	at javax.swing.GroupLayout.calculateAutopadding(Unknown Source)
	at javax.swing.GroupLayout.layoutContainer(Unknown Source)
	at java.awt.Container.layout(Unknown Source)
	at java.awt.Container.doLayout(Unknown Source)
	at java.awt.Container.validateTree(Unknown Source)
	at java.awt.Container.validateTree(Unknown Source)
	at java.awt.Container.validate(Unknown Source)
	at java.awt.Container.validateUnconditionally(Unknown Source)
	at java.awt.Dialog.conditionalShow(Unknown Source)
	at java.awt.Dialog.show(Unknown Source)
	at java.awt.Component.show(Unknown Source)
	at java.awt.Component.setVisible(Unknown Source)
	at java.awt.Window.setVisible(Unknown Source)
	at java.awt.Dialog.setVisible(Unknown Source)
	at us.deadcode.tab.FindAllBox.showFindBox(FindAllBox.java:366)
	at us.deadcode.tab.MainWindow.onFindAllMenu(MainWindow.java:218)
	at us.deadcode.tab.MainMenuBar$13.actionPerformed(MainMenuBar.java:338)
	at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
	at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
	at javax.swing.AbstractButton.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
````

**Comment by weisJ** [comment]
Please provide an example to reproduce.

**Comment by weisJ** [comment]
Probably fixed in 21780d96db7737ed72c6f3289eb4fe68619b43f7, but without an example to reproduce I can't check.

**Comment by Ilolka1337** [comment]
@weisJ work thx
